File: tasker-plugin/src/main/java/org/rm3l/router_companion/tasker/utils/FirebaseUtils.java
package org.rm3l.router_companion.tasker.utils;

import android.content.Context;
import androidx.annotation.NonNull;
import java.util.Objects;

public class FirebaseUtils {

  @NonNull
  public static String getFirebaseApiKey(@NonNull Context context) {
    return Objects.requireNonNull(ContextUtils.getConfigProperty(context, "FIREBASE_API_KEY", ""));
  }
}

File: app/src/main/java/org/rm3l/router_companion/utils/FirebaseUtils.kt
package org.rm3l.router_companion.utils

import android.content.Context
import org.rm3l.router_companion.utils.kotlin.getConfigProperty

class FirebaseUtils {

    companion object {

        @JvmStatic
        fun getFirebaseApiKey(context: Context) = context.getConfigProperty("FIREBASE_API_KEY", "")!!
    }
}
